prisma/migrations/0_init/migration.sql

```diff
@@ -347,7 +347,6 @@ CREATE TABLE "mentor" (
     "lastname" VARCHAR,
     "active" BOOLEAN NOT NULL DEFAULT true,
     "email" VARCHAR NOT NULL,
-    "verification" VARCHAR,
```

Comment:
Undo the changes to this file. It doesn't make sense to do this on the migration files.

Reply:
There are still some order/formatting changes. Could you also fix those?
Ideally, we leave this file untouched